packages/governance/src/contractHelper.js

```diff
@@ -102,12 +118,11 @@ const facetHelpers = (zcf, paramManager) => {
 
```

Comment:
The `ignoreContext` on like 94 above cannot be right. Probably you should double check all using of ignoreContext. We should only ever use it when defining a kind, not a far class.

Reply:
This doesn't seem to point to a line of code. I think this has been addressed.